modules/price/src/Plugin/Field/FieldFormatter/PriceCalculatedFormatter.php

```diff
@@ -84,11 +84,7 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
-    $currency_codes = [];
-    foreach ($items as $delta => $item) {
-      $currency_codes[] = $item->currency_code;
-    }
-    $currencies = $this->currencyStorage->loadMultiple($currency_codes);
+    $currencies = $this->currencyStorage->loadMultiple();
```

Comment:
This would cause all currencies to be loaded. Which generally should only be a few. But perhaps it'd be best to just move this to a single load call for the specific currency inside the `foreach` loop?